mypy/typeshed/stdlib/typing.pyi

```diff
@@ -435,29 +431,29 @@ class Mapping(Collection[_KT], Generic[_KT, _VT_co]):
     # TODO: We wish the key type could also be covariant, but that doesn't work,
     # see discussion in https://github.com/python/typing/pull/273.
     @abstractmethod
-    def __getitem__(self, k: _KT) -> _VT_co: ...
+    def __getitem__(self, __k: _KT) -> _VT_co: ...
     # Mixin methods
     @overload
     def get(self, key: _KT) -> _VT_co | None: ...
     @overload
-    def get(self, key: _KT, default: _VT_co | _T) -> _VT_co | _T: ...
+    def get(self, __key: _KT, __default: _VT_co | _T) -> _VT_co | _T: ...
```

Comment:
OK I see, this is the cause of #11646.

The `default=` argument might not apply to every `Mapping.get`, but it does work with `dict`. So maybe there should be special handling for `dict.get`?

Reply:
Doh. My bad, it `dict.get()` does not support keyword arguments, the original code was instead using `QueryDict` from Django.

So I suppose this should be solved in django-stubs instead.